astropy/coordinates/representation.py

```diff
@@ -897,9 +1159,9 @@ class UnitSphericalRepresentation(BaseRepresentation):
     def _dimensional_representation(cls):
         return SphericalRepresentation
 
-    def __init__(self, lon, lat, copy=True):
+    def __init__(self, lon, lat, differentials=None, copy=True):
         super(UnitSphericalRepresentation,
-              self).__init__(lon, lat, copy=copy)
+              self).__init__(lon, lat, copy=copy, differentials=differentials)
```

Comment:
Somehow prefer to have differentials before copy here too (I know, it doesn't matter).